os/kernel/semaphore/sem_tickwait.c

```diff
@@ -180,7 +180,7 @@ int sem_tickwait(FAR sem_t *sem, systime_t start, uint32_t delay)
 
 	ret = sem_wait(sem);
 	if (ret < 0) {
-		/* Return the errno from sem_wait() */
+		/* Return the errno from sem_wait() or from sem_timeout() */
```

Comment:
Anyway that is internal operation of sem_wait. and sem_tickwait gets a rerurn value of only sem_wait. sem_tickwait does not get a return value from sem_timeout. I don't understand why it is needed.